tests/test-frameworks/isolation/test

```diff
@@ -13,3 +13,7 @@ bazel build :classloader-shared
 ! bazel test :classloader-shared
 
 bazel test :process :process-2_11 :process-jvm
+
+bazel test --test_arg=--framework_args=-showtimes :none | grep 'be 2.*ms'
+bazel test --test_arg=--framework_args=-showtimes :classloader | grep 'be 2.*ms'
+bazel test --test_arg=--framework_args=-showtimes :process | grep 'be 2.*ms'
```

Comment:
Similar to some other tests that grep the output for specific strings, these tests tell specs2 to output timing data for tests and then grep for a specific string in the test output.